controllers/subscriber/subscriberControllers.js

```diff
@@ -111,16 +137,11 @@ export class EventSubscriberController extends BaseSubscriber {
 	}
 
 	hostConnected() {
-		// delay subscription otherwise import/upgrade order can cause selection mixin to miss event
-		this._subscriptionComplete = new Promise(resolve => {
-			requestAnimationFrame(() => {
-				this._subscribe();
-				resolve();
-			});
-		});
+		this._subscriptionComplete = this._keepTrying(() => this._subscribe(), 20, 100);
```

Comment:
One reason to keep these retry numbers different from the ID subscriber:

We *expect* the ID subscriber to fail, in certain conditions. For example pageable-observer uses two subscribers (event + ID), and when it's in ID mode we want the event subscriber to silently fail. So we don't want those expected failures to be too expensive.

The ID subscriber, on the other hand, would never get to its "keep-trying" stage if it's expecting to fail, because it only triggers when there's an explicit ID specified.

Reply:
I'm going to increase the window, I'm nervous about users with slow machines missing it. I'm thinking 400ms with a 40ms interval - 10 tries doesn't seem excessive.